### PR TITLE
Preliminary 6.2 

### PR DESCRIPTION
**Pull request by acharlieh**
Related to #52 We're changing things to make 6.2 install! Woot!

Proposed minor version bump.

### CODE REVIEW

**Comment by BobbyRyterski** [comment]
+1

**Comment by david-crowder** [comment]
+1

File: recipes/_configure_outputs.rb [hunk 1]
```diff
@@ -10,8 +10,8 @@
 if [:search_head, :forwarder, :cluster_master].include? node['splunk']['node_type']
   output_stanzas['tcpout'] = {
     'forwardedindex.0.whitelist' => '.*',
-    'forwardedindex.1.blacklist' => '_.*',
-    'forwardedindex.2.whitelist' => '(_audit|_internal)'
```
Comment (aaronblythe) [inline recipes/_configure_outputs.rb]: Why this change?  Can you provide links to the docs so I can learn more?

File: recipes/_install.rb [hunk 2]
```diff
@@ -73,6 +73,13 @@ def nsp
   mode '0700'
 end
 
+# SPL-89640 On upgrades, the permissions of this directory is set incorrectly.
```
Comment (aaronblythe) [inline recipes/_install.rb]: As too permissive or not permissive enough? (could be a slightly better comment if it answers this question).

**Comment by aaronblythe** [comment]
+1

File: recipes/_configure_outputs.rb (hunk 1 above)
Comment (acharlieh) [inline recipes/_configure_outputs.rb]: Added [in Splunk 6.1](http://docs.splunk.com/Documentation/Splunk/6.1/ReleaseNotes/MeetSplunk#Platform_instrumentation_framework) was the [Splunk Enterprise platform Instrumentation framework](http://docs.splunk.com/Documentation/Splunk/6.2.5/Troubleshooting/Abouttheplatforminstrumentationframework). This system by default on servers logs data to a new Splunk index called _introspection. As we're clustering indexers, we want to forward this data from search heads, and forwarders on which we enable this.

The way this is implemented as well (forward all indexes except for _thefishbucket, which isn't really an index but more of a scratch pad), additionally paves the way for the possibility of team separated  _* indexes, that way we could even start dividing up _internal data from forwarders for each team, and possibly lock down access to _internal itself.

**Comment by nimeshsubramanian** [comment]
+1

**Pull request merged**